seed/challenges/02-data-visualization-certification/react-projects.json

```diff
@@ -202,6 +265,25 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Рогулайк Данжеон-Кроулер",
```

Comment:
I understand your point. But I don't like how they 'translated' the word (actually, a term) `Roguelike`. I'd use `Roguelike-подобную` instead of `Рогулайк` like this: `Создайте Roguelike-подобную игру Подземелье`.